deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -49,18 +56,21 @@ monitoring:
   # externalMgrPrometheusPort: <port>
   # allow adding custom labels and annotations to the prometheus rule
   prometheusRule:
```

Comment:
Is it expected that these hierarchical settings don't have anything generated in the doc? Looks like it since the fully qualified name is used in the doc, but then we have info in the values.yaml that is not in the documentation.

Reply:
Any value in `values.yml` will be generated in the doc, but because it doesn't have `--` in the comment above, it will not have a descripton. I am reviewing the files to check this.

Reply:
For this in Particular, an entry will be generated for each of `prometheusRule.labels` and `prometheusRule.annotations`. Nothing will be generated for just `prometheusRule`. If we want to generate something for that, we'd need to add `--` to the comment above `prometheusRule` and remove it from `labels` and `annotatoons` below.

Current state: 
![image](https://user-images.githubusercontent.com/3438324/205149352-03a57c2f-498c-4602-8c0e-2bc73d506d95.png)

Could be changed to 
![image](https://user-images.githubusercontent.com/3438324/205149619-2f0ffff9-e958-4bb7-b8a1-bfda61f8b2ce.png)

Reply:
Thanks for the details, I'd vote for the current state instead of changing it.